src/content/h1/readme.md

```diff
@@ -87,7 +87,7 @@ In 2011, Halo: Combat Evolved Anniversary was released for Xbox 360. Often calle
 ## MCC (PC and Xbox One)
 Halo: The Master Chief collection (MCC) is actively maintained by [343 Industries][343i] for both PC and Xbox One. It brings the Halo series under a single [Game as a Service][gaas], including unified matchmaking and progression experiences. The PC port uses [Unreal Engine][unreal] as a menu and input layer over the respective engines of each included Halo game.
 
-It is possible to compile custom maps for MCC PC using Invader and some [experimental tooling][opack].
+It is possible to compile custom maps for MCC PC using older versions of Invader.
```

Comment:
broken link -- we don't have an opack page

Reply:
Shouldn't opack still be mentioned?

Reply:
No, the repo got deleted by its author and its obsolete with the stuff Zatarita is working on. Suppose I could mention that tool but don’t know enough about it to do it justice yet on the wiki

Reply:
Oh yeah sorry then. Forgot that was another project